app/styles/ui/check-runs/_ci-check-run-popover.scss

```diff
@@ -5,6 +5,10 @@
     width: 440px;
     top: 60;
     margin-left: -302px;
+
+    &::after {
+      border-bottom-color: var(--box-alt-background-color);
+    }
```

Comment:
While recording the video for this PR, I noticed the tip of the popover looked wrong on the dark theme, and this change fixes it:
![image](https://user-images.githubusercontent.com/1083228/160813203-287b7982-cdc1-44da-a4fd-21a66935c956.png)